plugins/jira-dashboard-backend/src/config.ts

```diff
@@ -22,11 +22,7 @@ export function resolveJiraToken(config: Config): string {
 }
 
 export function resolveUserEmailSuffix(config: Config): string {
-  try {
-    return config.getString(JIRA_USER_CONFIG_EMAIL_SUFFIX);
-  } catch (error) {
-    throw new Error(`Invalid Jira user path, ${error}`);
-  }
+  return config.getOptionalString(JIRA_USER_CONFIG_EMAIL_SUFFIX) || '';
```

Comment:
The config file needs to be updated since it is now optional: 

https://github.com/AxisCommunications/backstage-plugins/blob/main/plugins/jira-dashboard-backend/config.d.ts

Reply:
Done